packages/loader/container-loader/src/loader.ts

```diff
@@ -118,6 +120,28 @@ function createCachedResolver(resolver: IUrlResolver) {
     return cacheResolver;
 }
 
+/**
+ * Encapsulates a module entry point with corresponding code details.
+ */
+ export interface IFluidModuleWithDetails {
+    /** Code module to load. */
+    module: IFluidModule;
+    /** Optional. Code details associated with the module. */
+    details?: IFluidCodeDetails;
```

Comment:
>details [](http://example.com/codeflow?start=4&length=7)

this shouldn't be optional. the code loader can choose to just retun the one passed in

Reply:
Done.